code/modules/mob/living/carbon/human/human.dm

```diff
@@ -710,6 +710,15 @@
 	else
 		..()
 
+/mob/living/carbon/human/clear_cuffs(obj/item/I, cuff_break)
+	if(!I.loc || buckled)
+		return FALSE
+	if(I == wear_suit)
+		visible_message("<span class='danger'>[src] manages to [cuff_break ? "break" : "remove"] [I]!</span>")
+		to_chat(src, "<span class='notice'>You successfully [cuff_break ? "break" : "remove"] [I].</span>")
+		return TRUE
+	. = ..()

```

Comment:
Is there some reason this isn't called first? If there is use 
```suggestion
	return ..()
```
but you should really prefer to have the parent call happen first

Reply:
You're right, I was being dumb about this.